docs/data-collection.md

```diff
@@ -84,6 +84,24 @@ For the OpenShift 4 Developer Preview we will be sending back these exact attrib
   // subscription_sync_total is the number of times an OLM operator
   // Subscription has been synced, labelled by name and installed csv
   '{__name__="subscription_sync_total"}',
+  //
+  //OCS metrics to be collected:
+  // ceph_cluster_total_bytes gives the size of ceph cluster in bytes.
+  '{__name__="ceph_cluster_total_bytes"}',
+  // ceph_cluster_total_used_raw_bytes is the amount of ceph cluster storage used in bytes.
+  '{__name__="ceph_cluster_total_used_raw_bytes"}',
+  // ceph_health_status gives the ceph cluster health status
+  '{__name__="ceph_health_status"}',
+  // job:ceph_osd_metadata:count is the total count of osds.
+  '{__name__="job:ceph_osd_metadata:count"}',
+  // job:kube_pv:count is the total count pv's PV's present in OCP cluster.
```

Comment:
Please rephrase this description, I can’t quite understand what this metric accomplishes.

Reply:
Ack, Done.